fe1-web/src/features/events/functions/events.ts

```diff
@@ -23,8 +23,17 @@ export const categorizeEventsByTime = (time: Timestamp, events: EventState[]) =>
   const upcomingEvents: EventState[] = [];
 
   events.forEach((e: EventState) => {
-    // if end time is set, the event has ended
-    if (e.end) {
+    // ended events are either:
+    // - meetings that have ended more than {CURRENT_EVENTS_THRESHOLD_HOURS} hours ago
+    // - other events that have been marked as ended (e.end is set)
+    // TODO: should find a way to do this without relying on the event type
+    // for example, by making that events ends when organizers mark them as ended (same as the others)
+    if (
+      (e.eventType === 'MEETING' &&
+        e.end &&
+        e.end <= t - CURRENT_EVENTS_THRESHOLD_HOURS * 60 * 60) ||
```

Comment:
Probably just
```suggestion
        e.end <= t) ||
```
?

Current events is for events where the start time is in [now, now + CURRENT_EVENTS_THRESHOLD_HOURS], i.e. events that are about to start.